### PR TITLE
Invoicing edit

### PR DESCRIPTION
**Pull request by jdmooney**
Corrected typos and expanded some descriptions

### CODE REVIEW

File: userguide/subscription/includes/invoicing-usage.adoc [hunk 1]
```diff
@@ -133,7 +135,7 @@ If for some reason, there is some late usage reported in these past 2 periods, t
 
 Missing Usage Data:
 
-Currently it is considered an error to delete past usage data in these past 2 periods, the invoicing system would throw an error and prevent invoicing to happen correctly. If the late missing data is older than these past 2 periods, it is simply ignored.
+Currently it is considered an error to delete past usage data in these past 2 periods. IF this occurs, the invoicing system throws an error and prevents invoicing from happening correctly. If the late missing data is older than these past 2 periods, it is simply ignored.
```
Comment (pierre) [inline userguide/subscription/includes/invoicing-usage.adoc]: Typo: `IF`

File: userguide/subscription/includes/invoicing-usage.adoc [hunk 2]
```diff
@@ -89,18 +89,20 @@ In the aggregate mode, we will see one `USAGE` item with the aggregate amount ac
 }
 ----
 
-In this example, we see that we billed for two units `cell-phone-minutes` and `Mbytes`: For the `cell-phone-minutes`, the quantity recorded for the period was such, that we reached the maxium for tier 1 and started to bill some units for tier 2; for the `Mbytes` units, we billed  everything in the tier 1. The amount reported is the sum of the amount : 40 + 2000 + 100 = 2140.
+In this example, we have billed for two unit types: `cell-phone-minutes` and `Mbytes`. For `cell-phone-minutes`, the quantity recorded for the period reached the maximum for tier 1 and extended into tier 2. Since we are using the `ALL_TIER` policy, the amount for each tier is computed separately, giving 40 for tier 1 and 2000 for tier 2. For the `Mbytes` units, all usage remained in tier 1, so a single amount, 100, is computed. The amount reported is the sum of the individual amounts: 40 + 2000 + 100 = 2140.
+
+If we were instead using the `TOP_TIER` policy, all usage of `cell-phone-minutes` would be billed at the tier 2 price (20) for a total of 2400, so the aggregate amount reported would be 2500.
```
Comment (pierre) [inline userguide/subscription/includes/invoicing-usage.adoc:94]: It's great to have added an additional example!

**Review by pierre — CHANGES_REQUESTED** [review]

**Review by sbrossie — APPROVED** [review]
Looks good - we could merge after you have addressed the few comments from Pierre.

File: userguide/subscription/includes/invoicing-usage.adoc (hunk 1 above)
Comment (jdmooney) [reply userguide/subscription/includes/invoicing-usage.adoc]: corrected

**Pull request merged**